src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java21/InputStringTemplateNested.java

```diff
@@ -0,0 +1,18 @@
+//non-compiled with javac: Compilable with Java21
```

Comment:
Do we even have a CI test for Java 21? Also should this be `Java 21` (with space)?

Reply:
>Do we even have a CI test for Java 21?

Yes, this is always my first step when a new JDK comes out so that I can get the satisfaction of removing files that we can't parse on grammar updates :)

https://circleci.com/gh/checkstyle/checkstyle/538113

>Also should this be Java 21 (with space)?

It certainly could be, however this format was used long before I came along, so I have just followed it through the years.